### ISSUE TITLE
Tags and titles not blacklisting sidebar

### ISSUE THREAD

**Issue by LeetHaxor** [kwaschny/unwanted-twitch#114]
Categories and channels blacklisting works fine on sidebar channels. 

Tag and title blacklisting works fine on all other pages, just not on the sidebar.

**Comment by kwaschny** [comment]
The sidebar doesn't show title or tags and that's why they are not filtered. The extension can only block what is visible. It does not magically resolve information about the channel from somewhere else, like the API or what has been seen in a different view.

**Comment by LeetHaxor** [comment]
Stream titles are shown upon hovering the sidebar, so I logically deduced it would be possible to filter. Tags not being filterable makes sense, though.

**Comment by kwaschny** [comment]
True, the title is shown on hover, but I have no idea where that information comes from as it is not part of the DOM element. Let me know if you (or any other reader) have a selector to actually fetch that information.

**Comment by LeetHaxor** [comment]
I admittedly don't have extensive about coding/HTML, but I tried to figure out how the title data is being accessed on hover.

[This](https://i.imgur.com/9ngEUri.png) is what I was able to get from inspector. The highlighted line _is_ the title, but it disappears when not hovering, so I'm not sure it's really accessible otherwise. I was able to hide the title text by hiding `p.CoreText-sc-1txzju1-0.etFJDq`, but this hides ALL titles, so fetching specific titles just may not be possible.

Apologies for my lack of experience. It seems as though you were right with your initial thought and it's not possible, unless there's other means which I am not well-versed enough to figure out.  Thanks.
